### ISSUE TITLE
npm run rewrite fails

### ISSUE THREAD

**Issue by fergald** [jackbsteinberg/get-originals-rewriter#40]
Every attempt to run this on a very simple input file fails. It seems like it was broken by e35be3c94da44f230c6cf155d462b8380913ec05.

```
$@ npm run rewrite test2.js

> get-originals-rewriter@1.0.0-pre rewrite /home/fergal/github/jackbsteinberg/get-originals-rewriter
> jscodeshift -t ./lib/rewriter.js "test2.js"

Processing 1 files... 
Spawning 1 workers...
Sending 1 files to free worker...
/home/fergal/github/jackbsteinberg/get-originals-rewriter/node_modules/ts-morph/dist/codeBlockWriter/code-block-writer.js:13
tempThis["CodeBlockWriter"] = code_block_writer_1.default;
                            ^

TypeError: Cannot set property 'CodeBlockWriter' of undefined
    at Object.<anonymous> (/home/fergal/github/jackbsteinberg/get-originals-rewriter/node_modules/ts-morph/dist/codeBlockWriter/code-block-writer.js:8:1)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Module._compile (/home/fergal/github/jackbsteinberg/get-originals-rewriter/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Object.newLoader [as .js] (/home/fergal/github/jackbsteinberg/get-originals-rewriter/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
All done. 
Results: 
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 1.338seconds 
```

**Comment by domenic** [comment]
The best way to fix this is via #30.

**Comment by domenic** [comment]
This was "fixed" by https://github.com/jackbsteinberg/get-originals-rewriter/issues/40... but we kind of forgot to really fix it, so the party has moved to https://github.com/jackbsteinberg/get-originals-rewriter/issues/52.
